core/bytearray.go

```diff
@@ -51,3 +114,13 @@ func popcount(x byte) byte {
 	// which now contain the total count of set bits in the original byte
 	return (x + (x >> 4)) & 0x0F
 }
+
+// reverseByte reverses the order of bits in a single byte.
+func reverseByte(b byte) byte {
```

Comment:
Where is this function used? Do you plan to expose this as a feature?

Reply:
Not used anywhere, was planning to use it as a feature.

Reply:
Could you please create a ticket for this so we don't miss it.